### DISCUSSION TITLE
Solax RTU MODBUS starts failing after some time

### DISCUSSION THREAD

**Discussion by t-dan** [wills106/homeassistant-solax-modbus#1061] [Q&A] [answered]
Hallo,
I want to ask if anyone has an experience with the following:

I have a Solax inverter on RTU MODBUS through RS485 cable. It is not alone there; there are two more MODBUS devices on the same cable. There is a correct 120Ohm terminator. Communicating with Solax started without problems, and all data readings were correct (the most frequent readings are every 5secs). After two days or so, the readings began to report errors, which ended when no readings were available at all. 

What confuses me is that the last device on the wire (heat pump) still provides the correct data reading, so I suspect the wiring itself is correct.

Does anyone have a similar experience? I need a hint on where to look for an error ...

Thank you very much,
DT

**Comment by t-dan** [comment, marked as answer]
So it seems that the problem was in wiring (more specifically, the RJ45 connector needed to be screwed very tight).
Sorry for bothering you.

**Reply by wills106** [reply]
Glad you worked out what the issue was.
